sdk/resourcemanager/Azure.ResourceManager/docs/TestGuide.md

```diff
@@ -0,0 +1,97 @@
+# Test .NET management plane SDK
+
+## Frameworks
+
+We use [NUnit 3][nunit] as our testing framework.
+
+[Azure.Core.TestFramework's testing framework][core_tests] provides a set of reusable primitives that simplify writing tests for new Azure SDK libraries.
+
+## Testing project structure
+
+With the help of [Azure.ResourceManager.Template][mgmt_template], the basic testing project and file structure(see following for details) will be generated under `sdk\<service name>\<package name>\tests` directory.
+
+```text
+sdk\<service name>\<package name>\tests\Azure.ResourceManager.<service>.Tests.csproj
+sdk\<service name>\<package name>\tests\<service>ManagementTestBase.cs
+sdk\<service name>\<package name>\tests\<service>ManagementTestEnvironment.cs
+sdk\<service name>\<package name>\tests\Scenario
+sdk\<service name>\<package name>\tests\SessionRecords
+```
+
+**Note**: Considering that in Git directories exist implicitly, so you might need to create the `Scenario` and `SessionRecords` directories by yourself after cloning the repo.
```

Comment:
Do we really need to enforce this?
Maybe we need to document that the test code should go into `Scenario` directory

Reply:
let's keep it for now to make the test project structure within different RPs looks consistent as much as possible.